kong/status/init.lua

```diff
@@ -26,6 +26,7 @@ ngx.log(ngx.DEBUG, "Loading Status API endpoints")
 
 -- Load core health route
 api_helpers.attach_routes(app, require "kong.api.routes.health")
+api_helpers.attach_routes(app, require "kong.status.ready")
```

Comment:
There probably are going to be more APIs in the future that will be specific to Status API. I recommend renaming ready.lua to 'kong/api/status/init.lua' and then use appropriate import path here. Please note how the 'api' directory is in the path.

Reply:
good idea

Reply:
But we have to attach this route from kong/status/init.lua to make sure it is not accessible from Admin API.
Seems in our current architecture, `kong.api` namespace is for Admin API.